README.md

```diff
@@ -0,0 +1,54 @@
+# iThenticate Plagiarism Detector Plugin
+
+For OJS/OMP/OPS 3.x
+
+## Overview
+
+This plugin permits automatic submission of uploaded manuscripts to the [iThenticate service](http://www.ithenticate.com/) for plagiarism checking.
+1. You need an account of ithenticate.com (costs involved)
+   * paid via Crossref Similarity Check
+   * or, paid directly to iThenticate
+2. Install the plugin via the Plugin Gallery in the Dashboard
+3. Configure the plugin (see below)
+   * Enable the plugin via config.inc.php or in a specific journal/press/preprint context
+   * Configure the plugin with the username and password you get from ithenticate.com
+   * ![Example Settings configuration](ithenticate-settings.png)
+4. The author logs in and makes a submission
+   * The submission files will be sent to iThenticate in Step 4 of the submission process
+5. The Editor logs in to ithenticate.com to see the submission
+   * The submission will be found in a folder named by the Submission ID
+   * Click to see the report
+   * ![Example report review](ithenticate-report.png)
+
+Watch [the demo](https://www.ithenticate.com/demo) to know more about the features of iThenticate.
+
+## Configuration
+
+You may set the credentials in config.inc.php, or you may set the credentials per-journal in the plugin settings.  If credentials are present in config.inc.php, they will override those entered in the plugin settings form.
+
+The config.inc.php settings format is:
+
+```
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; iThenticate Plugin Settings ;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+[ithenticate]
+
+; Enable iThenticate to submit manuscripts after submit step 4
+;ithenticate = On
```

Comment:
Ah, I misread those -- when `ithenticate = On`, the manager *cannot* enable or disable the plugin. It's forced enabled.

But I think this will misbehave if users configure `ithenticate = On` without a main `username` and `password` -- which they are likely to try if they want to manage configuration in `config.inc.php` but only provide journal-specific configurations.

I might suggest removing the `ithenticate` setting entirely, and instead checking to see whether there is a defined username in order to determine whether the plugin should operate...?

Reply:
I'm always up for removing stuff from config.inc.php.

Reply:
Propose to address in a new PR.

Reply:
OK, but I think the "see README for installation instructions" locale key will still be necessary until this setting is removed. Otherwise people won't be able to enable the plugin and it won't be clear why.

Reply:
For those who already have this setting in config.inc.php, the plugin will continue to operate as before.  For those without this setting just installing the plugin, they can enable the plugin directly in the Website Settings without modifying config.inc.php.  The "See README" is obsolete as of this PR.

And, we'll make it even better with the next PR, which hopefully even precedes the next release.

Reply:
:+1: Sorry, I stumbled reading the same code the wrong way twice now. This looks good, thanks! I'll merge.

Reply:
A strong hint that the `getEnabled()` / `canEnable()` code needs to be better documented in the forthcoming PR!